dashboard/app/serializers/api/v1/pd/application_serializer.rb

```diff
@@ -14,4 +15,12 @@ def form_data
   def response_scores
     JSON.parse(object.response_scores || '{}')
   end
+
+  def meets_criteria
+    object.try(:meets_criteria) || nil
+  end
+
+  def bonus_points
+    object.try(:bonus_points) || nil
+  end
```

Comment:
The object being serialized can't be nil, can it? I don't think we need these 2 `try`s